tests/pos/i11185.scala

```diff
@@ -0,0 +1,5 @@
+class Test:
+  def foo(a: Int, b: Int) = a + b
+
+  Map(1 -> 2).map(foo _)
```

Comment:
Just a random thought, but if the collections ever get rewritten again `map` might not be overloaded anymore.

Reply:
Good point, I'll add another test with overloading.